thoth/python/source.py

```diff
@@ -405,3 +406,13 @@ def get_package_hashes(self, package_name: str, package_version: str, with_inclu
             result.append(doc)
 
         return result
+
+    def get_package_release_date(self, package_name: str, package_version: str,) -> datetime:
+        """Get time at which package was uploaded to package index."""
+        package_json = self._warehouse_get_api_package_info(package_name)
+        release = package_json["releases"][package_version]
+        artifact = next(x for x in release if x["python_version"] == "source")
+        if artifact is None:
+            raise ValueError(f"No source distribution found on {self.warehouse_api_url}")
+
+        return datetime.strptime(artifact["upload_time"], "%Y-%m-%dT%H:%M:%S")
```

Comment:
We have methods for manipulating with dates and datetimes in `thoth.common.helpers`. It helps to keep track of timezone issues and date/datetime formatting across the whole Thoth application.